playbooks/generic-deploy_ses_rook.yml

```diff
@@ -34,6 +34,14 @@
       tags:
         - bug
         - skip_ansible_lint
+- hosts: caasp-workers
```

Comment:
comment pointing to why or maybe the issue number/link?

Reply:
Good idea.